recipes/get_pangenes/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "20250123" %}
+{% set sha256 = "67665e4359dd16ae6fe1359c55a0a1d3eed07efa431c07a87c8c298a8bdbdde3" %}
+
+package:
+  name: get_pangenes
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+  script_env:
+   - LC_ALL=POSIX
+  run_exports:
+    - {{ pin_subpackage('get_pangenes', max_pin="x") }}
+
+source:
+  url: https://github.com/Ensembl/plant-scripts/archive/refs/tags/{{ version }}.tar.gz 
+  sha256: {{ sha256 }}
+
+requirements:
+  run:
+    - perl
+    - perl-db_file
+    #- minimap2 =2.17   
+    - minimap2 =2.24
+    - gffread =0.12.7  
```

Comment:
Are those strict pins really needed?

Reply:
Yes they are, those correspond to the versions benchmarked at https://github.com/Ensembl/plant-scripts/tree/master/pangenes 
For minimap2 in particular we observed differences with other versions